science/limesuite/Portfile

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.0
+PortGroup           cxx11 1.1
+PortGroup           github 1.0
+PortGroup           wxWidgets 1.0
+
+name                limesuite
+version             v19.01.0
+categories          science comms
+platforms           darwin macosx
+license             Apache-2
+maintainers         {@ra1nb0w irh.it:rainbow} openmaintainer
+description         provides drivers and SDR application support for the LMS7002M RFIC
+long_description    Lime Suite is a collection of software supporting several hardware \
+    platforms including the LimeSDR, drivers for the LMS7002M transceiver RFIC, and other \
+    tools for developing with LMS7-based hardware.
+homepage            https://myriadrf.org/projects/lime-suite/
+github.setup        myriadrf LimeSuite 025ffa1a1026a6307b001ea2a6ffb9ab5c4e8d63
```

Comment:
Please move the `github.setup` line to just below the `PortGroup           wxWidgets 1.0` line

Reply:
Created subports so moved the line into the right section